prow/cmd/deck/job_history.go

```diff
@@ -371,7 +371,7 @@ func getBuildData(ctx context.Context, bucket storageBucket, dir string) (buildD
 		logrus.WithError(err).Debugf("failed to read %s", prowv1.ProwJobFile)
 	} else {
 		if pj.Spec.Refs != nil && len(pj.Spec.Refs.Pulls) > 0 {
```

Comment:
do we still need `len(pj.Spec.Refs.Pulls) > 0 `?

Reply:
I believe you are right. Will fix.